benches/Cargo.toml

```diff
@@ -4,6 +4,19 @@ version = "0.1.0"
 edition = "2018"
 workspace = ".."
 
+[features]
+## Enables support for rust-libxml for benchmarks. rust-libxml requires native 
+## dependencies on the your local machine to run. 
+##
+## To run the benchmarks with the libxml feature on, first install the required
+## dependencies outlined in https://github.com/kwarc/rust-libxml/
+## You can run the benchmarks with the libxml feature on with:
+##
+## ```
+## $ cargo bench --features=libxml
+## ```
+libxml = ["dep:libxml"]
```

Comment:
I think this is incorrect. It should work without the explicit `libxml` feature. And you can move this comment to the `dependencies` section.

Reply:
Fixed with 788bcc8ae92cce1e1c940ab88b7da2966fd38a05